src/ui/common/src/components/pages/operator/id/hook.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react';
+import { useLocation, useParams } from 'react-router-dom';
+
+import { BreadcrumbLink } from '../../../../components/layouts/NavBar';
+import { DagResultResponse } from '../../../../handlers/responses/dag';
+import { OperatorResultResponse } from '../../../../handlers/responses/operator';
+import { WorkflowDagResultWithLoadingStatus } from '../../../../reducers/workflowDagResults';
+import { WorkflowDagWithLoadingStatus } from '../../../../reducers/workflowDags';
+
+export type useOperatorOutputs = {
+  breadcrumbs: BreadcrumbLink[];
+  operatorId: string;
+  operator: OperatorResultResponse;
+};
+
+export default function useOpeartor(
```

Comment:
handles operator redux retrieval

Reply:
nit: spelling is off here 

useOperator